jenkinsfile-runner-steward-image/scripts/run.sh

```diff
@@ -50,6 +50,10 @@ function main() {
   truncate -c -s 0 "${TERMINATION_LOG_PATH}"
   check_required_env_vars "${PARAM_VARS_MANDATORY[@]}"
 
+  # Fix for https://github.com/SAP/stewardci-core/issues/327
+  echo "Adding current directory (${PWD}) as git safe.directory"
```

Comment:
This is a low-level detail, not worth to be logged as an own step. Move this command _after_ line 57 (`echo "Cloning ..."`).